line 5 unreachable code: SUCCESS
line 6 attempt to compute `_1 * move _13`, which would overflow: SUCCESS
line 8 assertion failed: z == 6: SUCCESS
